### ISSUE TITLE
Remove comma-dangle

### ISSUE THREAD

**Issue by cseas** [iamturns/eslint-config-airbnb-typescript#311]
typescript-eslint strongly recommends against using the "comma-dangle" rule:
https://typescript-eslint.io/rules/comma-dangle/

Since it is part of the original airbnb config, maybe we should just let the eslint one stay as it is instead and remove the override with the equivalent typescript-eslint rule.

**Comment by Kenneth-Sills** [comment]
Yup! With #354, since the Typescript rule was outright removed, this rule is being migrated to [ESLint Stylistic](https://eslint.style/). Once the upstream Airbnb config is updated, the override will be removed entirely.
